.github/CODEOWNERS

```diff
@@ -1,9 +1,8 @@
 # Documentation
-/docs                                           @ajitbohra
-/docs/designers-developers/developers           @mkaz @ajitbohra
-/docs/designers-developers/designers            @mkaz @ajitbohra
+/docs                                           @ajitbohra @mkaz
 
 # Schemas
+/schemas                                        @mkaz
 /schemas/json                                   @ajlende
```

Comment:
@mkaz, as far as I remember the way it works it won’t notify you for the subfolders that Alex created. You probably need to put your name in both entries. I’m happy to be listed for block.json but it might be too granular setting.